contracts/LiquidityConversionRates.sol

```diff
@@ -3,55 +3,136 @@ pragma solidity ^0.4.18;
 import "./ConversionRates.sol";
 import "./LiquidityFormula.sol";
 
-contract LiquidityConversionRates is ConversionRates, LiquidityFormula {
+contract LiquidityConversionRates is ConversionRatesInterface, LiquidityFormula, Withdrawable, Utils {
+    ERC20 token;
     uint public r;
     uint public Pmin;
-    uint public precision;
+    uint public qtyPrecision;
```

Comment:
it is not exactly qty, as r and Pmin are not qty. Please try to think of better name. Maybe `formulaPrecision`?

Reply:
using formulaPrecision..